<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartDeviceManager</class>
 <widget class="QMainWindow" name="SmartDeviceManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>656</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SmartDeviceManager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="bodyContainer" native="true">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>771</width>
      <height>331</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Thông Tin Thiết Bị</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>13</y>
       <width>172</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Danh Sách Thiết Bị</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>500</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPlainTextEdit" name="txtDevInfo">
     <property name="geometry">
      <rect>
       <x>237</x>
       <y>39</y>
       <width>521</width>
       <height>281</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="controlContainer" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>781</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btnAddDevice">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Thêm Thiết Bị</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRemoveDevice">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Xóa Thiết Bị</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLightSwitch">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Bật/Tắt</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="logWidget" native="true">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>417</y>
      <width>778</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QPlainTextEdit" name="txtLogging">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>761</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Logging</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
